content/spin/serverless-ai-tutorial.md

```diff
@@ -623,12 +631,11 @@ route = "/internal/kv-explorer/..."
 - b) have configured your `spin.toml` file to explicitly configure the model (as shown above), and
 - c) have [signed up for the Serverless AI private beta](https://developer.fermyon.com/cloud/serverless-ai).
 
-Now let's build and deploy our Spin Application locally. Run the following command to build your application: 
+Now, let's build and run our Spin Application locally. (***Note:** If you are following along with the TypeScript/JavaScript example, you will first need to run `npm install`. Otherwise, please continue to the following `spin` command.):
```

Comment:
if you keep it as is

```suggestion
Now, let's build and run our Spin Application locally. (**Note:** If you are following along with the TypeScript/JavaScript example, you will first need to run `npm install`. Otherwise, please continue to the following `spin` command.)
```

(I know about the preference for colons but it's not grammatical to have one after the end of a sentence and I will sustain mild injury on this hill.)